index.php

```diff
@@ -115,6 +147,50 @@
     <br>
     <br>
     <div class="container">
+      <div id="jmsToggle" class="row">
+        <div class="col-md-8 col-md-offset-2">
+          <ul class="list-group">
+              <li class='list-group-item'>
+                <div class='form-group'>
+                  <button class='ml btn btn-primary' id="toggleForm">
+                    <span>Show JMS Form</span>
+                  </button>
+                  <?php
+                    if ( isset ( $_GET['amqMsg'] ) && $_GET['amqMsg'] == "success" ) {
+                      echo "  <span>Message sent successfully</span>\n";
+                    }
+                  ?>
+                </div>
+              </form>
+              </li>
+          </ul>
+        </div>
+      </div>
+      <div id="jmsForm" class="row" style="display:none">
+        <div class="col-md-8 col-md-offset-2">
+          <ul class="list-group">
+              <li class='list-group-item'>
+                <form class='form-inline' method='post'>
+                <div class='form-group'>
+                  <textarea class='ml form-control' rows='6' name='message'><?php
+echo "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n";
```

Comment:
do we need to echo this, seems like it could be just as well included inline?

if we do, perhaps we can have this in a constant? something like: https://secure.php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc

Reply:
Yes, much better.